packages/shared/src/graphql/squads.ts

```diff
@@ -0,0 +1,14 @@
+import { Source } from './sources';
+
+export interface Squad extends Source {
+  active: boolean;
+  handle: string;
+  image: string | null;
```

Comment:
This cannot be null

Reply:
If it is empty in the db, it returns null currently. Thats why I set it like this

Reply:
You're right! Eventually, I didn't set it to a required field. Sorry!